error-prone-contrib/src/test/resources/tech/picnic/errorprone/refasterrules/MapRulesTestInput.java

```diff
@@ -11,7 +13,7 @@
 final class MapRulesTest implements RefasterRuleCollectionTestCase {
   @Override
   public ImmutableSet<?> elidedTypesAndStaticImports() {
-    return ImmutableSet.of(HashMap.class);
+    return ImmutableSet.of(HashMap.class, requireNonNullElse("foo", "bar"));
```

Comment:
I notice that in `elidedTypesAndStaticImports` we use `null` for non-primitive parameters (the code isn't executed anyway), so let's stick with that.

(That said, perhaps we _should_ find a way to run these files, and enforce more realistic test cases that way... something to ponder.)